client/landing/stepper/declarative-flow/internals/steps-repository/design-choices/index.tsx

```diff
@@ -91,66 +68,17 @@ const DesignChoicesStep: StepType< { submits: { destination: string } } > = ( {
 		submit?.( { destination } );
 	};
 
-	const getCreateWithAILabel = () => {
-		if ( hasEnTranslation( 'Create with AI {{small}}(BETA){{/small}}' ) ) {
-			return translate( 'Create with AI {{small}}(BETA){{/small}}', {
-				components: {
-					small: <span className="design-choices__beta-label" />,
-				},
-			} );
-		}
-
-		if ( hasEnTranslation( 'Create with AI' ) ) {
-			return translate( '%s {{small}}(BETA){{/small}}', {
-				args: [ translate( 'Create with AI' ) ],
-				components: {
-					small: <span className="design-choices__beta-label" />,
-				},
-				comment: 'Do not translate',
-			} );
-		}
-
-		return translate( 'Create with AI (BETA)' );
-	};
```

Comment:
We're planning to reuse this, since the new screen is better than the previous one. [Here's the task](https://github.com/Automattic/wp-calypso/issues/102252). 

Could you please leave the code for rendering the screen, or if you have the bandwidth, also implement those changes? In that case, I'd suggest a follow-up PR anyway.

Reply:
Yeah I was thinking the same here. The new choice screen design looks better so I'm thinking this might not be something we want to remove.